README.md

```diff
@@ -5,47 +5,134 @@
+| Working with External Files    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/working-with-external-files.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Integrationg with Label Studio for Annotations    | <a target="_blank" href="https://pixeltable.readme.io/docs/label-studio"> <img src="https://img.shields.io/badge/Docs-Label Studio-blue" alt="Visit our documentation"/></a>
 
+## 🧱 Code Samples
+
+### All media data (videos, images, audio) resides in external files, and Pixeltable stores references to those
 ```python
-import pixeltable as pxt
-pxt.init()
+# The files can be local or remote (e.g., in S3). For the latter, Pixeltable automatically caches the files locally on access.
+v = pxt.create_table('external_data.videos', {'video': pxt.VideoType()})
+
+prefix = 's3://multimedia-commons/'
+paths = [
+    'data/videos/mp4/ffe/ffb/ffeffbef41bbc269810b2a1a888de.mp4',
+    'data/videos/mp4/ffe/feb/ffefebb41485539f964760e6115fbc44.mp4',
+    'data/videos/mp4/ffe/f73/ffef7384d698b5f70d411c696247169.mp4'
+]
+v.insert({'video': prefix + p} for p in paths)
 ```
-> [!NOTE]
-> Check out the [Pixeltable Basics](https://pixeltable.readme.io/docs/pixeltable-basics) tutorial for a tour of its most important features.
+Learn more on how to [access the original file paths and work with your data](https://pixeltable.readme.io/docs/working-with-external-files).
 
-## 💡 Get Started
-Learn how to create tables, populate them with data, and enhance them with built-in or user-defined transformations and AI operations.
+### Create a computed column based on a function calling an object detection model
+```python
+# Pixeltable is downloading the model from Huggingface (if necessary), instantiating it, and caching it for later use
+t['detections'] = huggingface.detr_for_object_detection(t.input_image, model_id='facebook/detr-resnet-50')
+```
+Learn more on how to use computed colums through the example of [comparing object detection models](https://pixeltable.readme.io/docs/object-detection-in-videos).
+
+### Build User-Defined Functions to expand Pixeltable
+```python
+# The longest_word Python function isn't a Pixeltable UDF (yet); Adding the decorator turns it into a UDF:
+@pxt.udf
+def longest_word(sentence: str, strip_punctuation: bool = False) -> str:
+    words = sentence.split()
+    if strip_punctuation:  # Remove non-alphanumeric characters from each word
+        words = [''.join(filter(str.isalnum, word)) for word in words]
+    i = np.argmax([len(word) for word in words])
+    return words[i]
+```
+Learn more on how to leverage [UDFs](https://pixeltable.readme.io/docs/user-defined-functions-udfs)
```

Comment:
As above (and in all these examples) I prefer to use the exact title of the tutorial for the link, as in:

```
Learn more about user-defined functions: [UDFs in Pixeltable](...)
```